values.yaml

```diff
@@ -30,20 +30,22 @@ backend:
     pullPolicy: Always
   controlPlaneJSON: false
   ingress:
-    enabled: false
+    enabled: true
```

Comment:
i don't think defaulting to a GKE specific ingress enabled is the right choice here, although i agree that it should be supported.